src/backend/replication/walsender.c

```diff
@@ -3873,20 +3873,23 @@ backpressure_lag(void)
 			LSN_FORMAT_ARGS(applyPtr));
 
 		if ((writePtr != UnknownXLogRecPtr
+			&& writePtr != InvalidXLogRecPtr
```

Comment:
When do these feedback fields can be `InvalidXLogRecPtr`?
Is it just a moment before safekeeper received some values from pageserver?